scripts/setup/pgroonga-config

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -eux
+
+dbversion=$(crudini --get /etc/zulip/zulip.conf postgresql version)
+dbname=$(crudini --get /etc/zulip/zulip.conf postgresql database_name 2>/dev/null || echo zulip)
+dbuser=$(crudini --get /etc/zulip/zulip.conf postgresql database_user 2>/dev/null || echo zulip)
+
+sharedir="${1:-/usr/share/postgresql/$dbversion}"
+applied_file="$sharedir/pgroonga_setup.sql.applied"
+
+installed_version=$(dpkg-query --show --showformat='${Version}' "postgresql-$dbversion-pgdg-pgroonga")
+if [ ! -f "$applied_file" ]; then
+    sql="CREATE EXTENSION PGROONGA; GRANT USAGE ON SCHEMA pgroonga TO $dbuser"
```

Comment:
```suggestion
    sql="CREATE EXTENSION PGROONGA"
```

We don't need `GRANT USAGE ON SCHEMA pgroonga` because it's only for old PGroonga (PGroonga 1.x). Newly installed case doesn't need to care about it.

And `pgroonga` schema will be removed in PGroonga 4.x (not planed yet). If we have this, this script will be failed with PGroonga 4.x.

Reply:
Ah, thanks for that catch!  Added a commit to drop it from the other places as well.

Reply:
Ah, we have `scripts/setup/create-pgroonga.sql`.
How about using it instead of writing the same SQL here?

Reply:
@alexmv It seems that `; GRANT USAGE ON SCHEMA pgroonga TO $dbuser` still exists. We may need a follow-up PR that removes it.

Reply:
Gah -- great catch, thanks.  I removed it in the first commit only to bring it back in the second because I didn't update it.

Reply:
And `scripts/setup/create-pgroonga.sql` hardcodes the `\connect zulip`, so isn't reusable -- it's used for the PostgreSQL Docker image.